server/odc-service/src/main/java/com/oceanbase/odc/service/task/model/DatabaseSequence.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2023 OceanBase.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.oceanbase.odc.service.task.model;
+
+public class DatabaseSequence {
+
+    private Long id;
+
+    private String name;
```

Comment:
id will be fine, name should be queried out so that suggest that this class may be removed